geometry_msgs/Pose2D target

---

std_msgs/Bool result